Finale/Finale/Presentation/TrainingBook/TrainingBookView.swift

```diff
@@ -79,6 +79,15 @@ struct TrainingBookView: View {
 
                 QuizView(
                     quiz: $viewModel.quiz,
+                    submitAction: {
+                        withAnimation(.easeInOut) { presentedQuiz = false }
+                        viewModel.makeTrainingResult()
+                        if let result = viewModel.trainingResult {
```

Comment:
P1: 아래 처럼 send 통해서 가야할 것 같아요
```suggestion
                        withAnimation(.easeInOut) { presentedQuiz = false }
                        viewModel.send(.submitResult)
                        if let result = viewModel.trainingResult {
```